docker-compose-test.yml

```diff
@@ -150,9 +150,11 @@ services:
   postgres:
     container_name: postgres
     environment:
-      POSTGRES_DB: postgres
-      POSTGRES_PASSWORD: postgres
-      POSTGRES_USER: postgres
+      AERIE_USERNAME: '${AERIE_USERNAME}'
+      AERIE_PASSWORD: '${AERIE_PASSWORD}'
```

Comment:
Almost. AERIE_PASSWORD should be before AERIE_USERNAME :)

Reply:
Gah! Okay, _now_ it should be good. :-)